templates/org/milestones/view.tmpl

```diff
@@ -0,0 +1,61 @@
+{{template "base/head" .}}
+<div role="main" aria-label="{{.Title}}" class="page-content organization milestone">
+	{{template "org/header" .}}
+	<div class="ui container">
+		<div class="gt-df">
```

Comment:
These `gt-` classes no longer exist so this is surely broken, please migrate to Tailwind.css's `tw-` classes.